modin/engines/base/io/io.py

```diff
@@ -11,41 +11,35 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
-"""This module houses `BaseIO` class, that contains basic utils
-and default implementation of IO functions.
+"""
+Module houses `BaseIO` class.
 
+`BaseIO` is base class for IO classes, that stores IO functions.
 """
 
 import pandas
-from pandas.util._decorators import Appender
+from pandas.util._decorators import doc
 from collections import OrderedDict
 from modin.error_message import ErrorMessage
 from modin.backends.base.query_compiler import BaseQueryCompiler
 from typing import Optional
 
-_doc_returns_qc = """BaseQueryCompiler:
-    QueryCompiler with read data.
-
-"""
-
-_doc_returns_qc_or_parser = """BaseQueryCompiler or TextParser:
-    QueryCompiler or TextParser with read data.
-
-"""
-
-_doc_returns_excel = """BaseQueryCompiler, dict or OrderedDict:
-    QueryCompiler or OrderedDict/dict with read data.
-
-"""
+_doc_default_io_method = """
+{summary} using Pandas.
 
-_doc_default_io_method = """{summary} using Pandas.
 For parameters description please refer to Pandas API.
 
 Returns
 -------
 {returns}
 """
 
+_doc_returns_qc = """BaseQueryCompiler :
+    QueryCompiler with read data."""
+
+_doc_returns_qc_or_parser = """{BaseQueryCompiler, TextParser} :
```

Comment:
```suggestion
_doc_returns_qc = """BaseQueryCompiler
    QueryCompiler with read data."""

_doc_returns_qc_or_parser = """{BaseQueryCompiler, TextParser}
```
I don't think we need colons after return types, do we?